src/main/java/uk/gov/ch/repository/update/trusts/CorporateTrusteesRepository.java

```diff
@@ -7,14 +7,45 @@
 
 public interface CorporateTrusteesRepository extends
         PagingAndSortingRepository<CorporateTrusteeData, Long> {
-    @Query(value = "SELECT ctd.trustee_id, ctd.trustee_name, "
-            + "ctd.on_register_in_country_formed, ctd.register_location, ctd.registration_number, "
-            + "ctd.law_governed, ctd.legal_form, ctd.country, "
-            + "t.corporate_ind, t.trustee_type_id, "
-            + "t.appointment_date, t.ceased_date "
+    @Query(value = "SELECT t.trustee_id, "
+            + "       t.trustee_type_id, "
+            + "       t.appointment_date, "
+            + "       t.ceased_date, "
+            + "       t.corporate_ind, "
+            + "       ctd.trustee_name, "
+            + "       ctd.register_location, "
+            + "       ctd.registration_number, "
+            + "       ctd.law_governed, "
+            + "       ctd.legal_form, "
+            + "       ctd.country, "
+            + "       ctd.on_register_in_country_formed, "
+            + "       ctd.service_same_as_roa_ind, "
+            + "       tal_oa.appointment_address_type_id, "
```

Comment:
We don't actually need tal_oa.appointment_address_type_id selected. I think we need all the others but if you see any that are not then good to remove.